gulp-tasks/config.js

```diff
@@ -17,6 +17,7 @@ let config = {
 config.buildFolderRoot = path.join('dist', config.artifactPlatform + '-' + config.artifactArch );
 config.buildFileNamePrefix = config.artifactName;
 config.buildFolderPath = path.resolve(config.buildFolderRoot);
+config.genFolderPath = path.join(config.buildFolderPath, 'devsuite-win32-x64');
```

Comment:
Right, I was trying to skip app generation every time I run the build and forget to clean up. 
Another reason for it not to be here is it is windows specific.
I'll remove it.
Thanks @jrichter1!